release/air_tests/air_benchmarks/compute_gpu_1.yaml

```diff
@@ -0,0 +1,15 @@
+cloud_id: {{env["ANYSCALE_CLOUD_ID"]}}
+region: us-west-2
+
+max_workers: 0
+
+head_node_type:
+    name: head_node
+    instance_type: g3.8xlarge
+
+worker_node_types:
```

Comment:
Remove if 0 workers?

Reply:
i tried to request without this field but it's required 

```
HTTP response body: {"error":{"detail":[{"loc":["body","config","worker_node_types"],"msg":"field required","type":"value_error.missing"}],"body":{"name":"oss-release-tests_mWEC__compute__air_benchmark_pytorch_training_e2e_gpu_1x1__b495cab4ae6fcac8c3bde264dd55fe27a1a23f024d930625bfd5f0380dc8852f","project_id":"prj_mWECugke9RzMh79BZQqeykjN","config":{"cloud_id":"cld_401TPoxgB8MM6A0NNQauOV","region":"us-west-2","max_workers":0,"head_node_type":{"name":"head_node","instance_type":"g3.8xlarge"}}},

"message":"body.config.worker_node_types: field required"}}
```